src/app/pages/MediaAssetPage/MediaMessage/index.jsx

```diff
@@ -15,6 +15,10 @@ import {
 import { MediaMessage } from '@bbc/psammead-media-player';
 import { ServiceContext } from '#contexts/ServiceContext';
 import { GridItemConstrainedLarge } from '#lib/styledGrid';
+import nodeLogger from '#lib/logger.node';
+import logCodes from '#lib/logger.const';
```

Comment:
And then can we just import the codes that we need?

e.g. import { NO_TRANSLATION_FOUND } from '#lib/logger.const';

Reply:
👍